.github/workflows/check-yaml.yml

```diff
@@ -0,0 +1,22 @@
+on:
+  pull_request:
+    paths:
+      - '.github/actions/*/*.yml'
+      - '.github/workflows/*.yml'
+      - 'templates/.github/actions/*/*.yml'
+      - 'templates/.github/workflows/*.yml'
+
+name: Check YAML
+
+jobs:
+  check-yaml:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: galargh/validate-yaml-schema@0be36a8865108d9fc0b2cae260dc0f9f649fac96 # forked v1.0.3
```

Comment:
why was this forked? if a change was necessary, perhaps we can send a PR and phase out the fork sooner than later.

Reply:
It's because it didn't work for YAMLs nested more than one level deep. Of course, I've already created a PR - https://github.com/nwisbeta/validate-yaml-schema/pull/20

Reply:
SGTM - add a TODO with the link perhaps, so we remember to update when merged